admin_auto_filters/filters.py

```diff
@@ -36,17 +27,17 @@ class Media:
         }
 
     def __init__(self, request, params, model, model_admin):
-        self.parameter_name = '{}__{}__exact'.format(self.field_name, self.field_pk)
+        self.parameter_name = '{}__{}__in'.format(
+            self.field_name, self.field_pk)
         super().__init__(request, params, model, model_admin)
 
         if self.rel_model:
             model = self.rel_model
 
         remote_field = model._meta.get_field(self.field_name).remote_field
 
-        widget = AutocompleteSelect(remote_field,
-                                    model_admin.admin_site,
-                                    custom_url=self.get_autocomplete_url(request, model_admin),)
+        widget = AutocompleteSelectMultiple(
```

Comment:
we need  to provide the functionality for custom url  for autocomplete, but it would be completely removed in this change. Require changes here.

Reply:
Should be done in this PR: https://github.com/farhan0581/django-admin-autocomplete-filter/pull/81